### PR TITLE
Add readme.txt file to open on nuget install

### PR DESCRIPTION
**Pull request by martijn00**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)

Feature

### :new: What is the new behavior (if this is a feature change)?

This softens the pain of not having a starterpack, or install nuget files.

### :boom: Does this PR introduce a breaking change?

No

### :thinking: Checklist before submitting

- [x] All projects build
- [x] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [x] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contribute/mvvmcross-docs-style-guide))
- [x] Rebased onto current develop

### CODE REVIEW

File: MvvmCross/MvvmCross.csproj [hunk 1]
```diff
@@ -74,6 +74,10 @@
     <Compile Include="Platform\Xamarin-common\**\*.cs" />
     <PackageReference Include="Tizen.NET" Version="4.0.0" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <None Include="..\readme.txt" pack="true" PackagePath="." />
```
Comment (nickrandolph) [inline MvvmCross/MvvmCross.csproj]: @martijn00 is there a reason why the readme is up a folder level? is this just so it doesn't get included as source code?

Comment (martijn00) [reply MvvmCross/MvvmCross.csproj]: I thought to add it as solution item, so it could work for every project. I could move it into the project itself.

Comment (nickrandolph) [reply MvvmCross/MvvmCross.csproj]: Given that it has a specific list of files I think move to project and then we may look at having an additional readme for mvvmcross.forms

**Comment by martijn00** [comment]
@nickrandolph Feedback is implemented

File: Projects/Playground/Playground.Forms.iOS/Setup.cs [hunk 2]
```diff
@@ -26,12 +26,5 @@ protected override IEnumerable<Assembly> GetViewAssemblies()
         protected override MvxFormsApplication CreateFormsApplication() => new FormsApp();
 
         protected override IMvxApplication CreateApp() => new Core.App();
-
```
Comment (nickrandolph) [inline Projects/Playground/Playground.Forms.iOS/Setup.cs]: @martijn00 was this intentionally added to this PR?

**Pull request merged**